src/features/in-app-messenger/types/template-params.js

```diff
@@ -18,3 +18,12 @@ export type PreviewTitleBodyTwoButtonsPopoutParams = {|
     title: string,
     ...PreviewParams,
 |};
+
+export type PreviewTitleBodyTwoButtonsModalParams = {|
```

Comment:
This looks the same as the props above, can we use a generic type for both? maybe named `PreviewTitleBodyTwoButtonsParams`

Reply:
Good point, it had different props when I was creating it on the beginning, but after a refactor it became the same. Good catch!